tools/build.py

```diff
@@ -3,10 +3,13 @@
 import os
 import sys
 import subprocess
+import struct
+import platform
 
 ######## GLOBALS #########
 MAINPREFIX = "z"
 PREFIX = "tfar_"
+USEARMAKE = False
```

Comment:
If armake works fine we can use it on our CI. Then I can remove my special mikero setup crap.

Reply:
currently the windows-version has a bug, that's why I've added this.